docs/reporting.rst

```diff
@@ -1740,19 +1796,456 @@ Key specifications:
   or “HeadtoHelp / HeadtoHealth”)
 * The report counts intakes, not dispatches
 
-.. _produce-twb-report:
+.. _category-m:
+
+Category M: Monitoring service delivery
+---------------------------------------
+
+These reports have been introduced to monitor MMHCs, however all except for the
+:ref:`category-m6` are pertinent for other program types as well.
+
+.. contents:: Available category M reports
+   :local:
+   :depth: 1
+
+.. _category-m1:
+
+M1 — Client wait time
++++++++++++++++++++++
+
+Wait time defined as number of days between a client’s intake referral date 
+and the date of the client’s first service contact.
+
+Key specifications:
+
+* Service Contacts flagged as ‘No Show’ are not included for this purpose.
+* A client’s wait time is defined as the time from intake referral to the
+  first attended contact
+
+.. _category-m2:
+
+M2 — Activity Reports by Age group
+++++++++++++++++++++++++++++++++++
+
+Number and proportion of active clients, active episodes and service contacts by age.
+
+Key specifications:
+
+* Only ‘active clients’ are reported. A Client is defined as ‘active’ and in
+  scope for inclusion in this report if they had one or more active episodes
+  in the reporting period, where ‘active episode’ is defined below.
+* Only ‘active episodes’ are reported. An Episode is defined as ‘active’ and
+  in scope for inclusion in this report if it had one or more Service Contacts
+  recorded in the period. No distinction is made between Open and Completed
+  Episodes
```

Comment:
Can we link to [this active episode definition](https://docs.pmhc-mds.com/projects/data-specification/en/latest/data-model-and-specifications.html#active-episode)?